lua/plenary/popup/init.lua

```diff
@@ -115,6 +115,10 @@ end
 function popup.create(what, vim_options)
   vim_options = vim.deepcopy(vim_options)
 
+  if vim.o.modifiable == false then
+    vim.o.modifiable = true
+  end
```

Comment:
this just enables this globally. change this setting, we probably dont want this?!